config/pod.yaml.hbs

```diff
@@ -128,6 +128,16 @@ spec:
     - mountPath: /opt/sdeventcache
       name: sd-event-cache
     {{/if}}
+  env:
+    {{#if secret_entity}}
+      {{#each volume_mounts}}
+      - name: {{this.env_name}}
```

Comment:
can we add `readOnly: true` to the volume

Reply:
also, why is it looping `volume_mounts`? The keys for `volume_mounts` property are `path`, `type` and `name`